pubnub/endpoints/pubsub/publish.py

```diff
@@ -56,11 +56,15 @@ def ttl(self, ttl):
 
     def build_data(self):
         if self._use_post is True:
-            cipher = self.pubnub.config.cipher_key
-            if cipher is not None:
-                return '"' + self.pubnub.config.crypto.encrypt(cipher, utils.write_value_as_string(self._message)) + '"'
-            else:
-                return utils.write_value_as_string(self._message)
+            stringified_message = utils.write_value_as_string(self._message)
+
+            if self.pubnub.config.crypto_module:
+                stringified_message = self.pubnub.config.crypto_module.encrypt(stringified_message)
+            elif self.pubnub.config.cipher_key is not None:  # The legacy way
+                stringified_message = '"' + self.pubnub.config.crypto.encrypt(
```

Comment:
seems risky to keep doing '"' + x + '"' everywhere, are you sure x can't contain a " character? also here above in line 62 you don't add the " - is it required? I'm just pointing it out because it's different than in other places